egs/demask/README.md

```diff
@@ -0,0 +1,219 @@
+<div align="center">
+[Inspiration](#inspiration) •
+[What it does](#what-it-does) •
+[How we built it](#how-we-built-it) •
+[Asteroid highlights](#asteroid-highlights) •
+[Challenges and Accomplishments](#challenges-we-ran-into) •
+[Acknowledgement](#acknowledgement)
+</div>
+
+## Inspiration
+
+#### Asteroid's inspiration
+It all started during a speech processing research project.
+We wanted to go fast and tried "ready-to-use" speech separation/speech enhancement models.
+We quickly realized that nothing was really working as expected,
+and spent our time fixing other people's bugs instead of doing research.
+This is the struggle that inspired Asteroid, and motivated us to open-source our
+code with things that _just work_.
+
+While sharing research code is already a step in the right direction,
+sharing readable and reproducible code should be the standard.
+Asteroid aims at empowering developers and researchers with tools that makes this even easier.
+
+#### DeMask's inspiration
+
+We were having a fast call and one of us was in the train.
+We couldn't hear him well because he was wearing a mask.
+We directly thought about building surgically masked speech enhancement model with
+Asteroid and we were on it !
```

Comment:
i think it's a bit short. Maybe add a few more buzz words like covid, pandemic etc

Reply:
Feel free to make suggestions, please